protoc-builder/Dockerfile.go

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.19-bullseye AS go-builder
```

Comment:
1.19 is the correct one?

Reply:
yup:

https://github.com/namely/docker-protoc/blob/075b66194ad47c4735916ddbb0739e9910791e0f/Dockerfile#L20

https://github.com/namely/docker-protoc/blob/075b66194ad47c4735916ddbb0739e9910791e0f/variables.sh#L10